modules/apps/change-tracking/change-tracking-service/src/main/java/com/liferay/change/tracking/model/impl/CTCollectionImpl.java

```diff
@@ -27,6 +32,24 @@ public String getUserName() {
 		return user.getFullName();
 	}
 
+	@Override
+	public boolean isEmpty() {
+		long ctCollectionId = getCtCollectionId();
+
+		int ctEntriesCount =
+			CTEntryLocalServiceUtil.getCTCollectionCTEntriesCount(
+				ctCollectionId);
```

Comment:
Do an early return if `ctEntriesCount` is not 0. That way we can save a call to `getCTMappingTableInfos`.